examples/step-3/doc/results.dox

```diff
@@ -308,6 +308,7 @@ data_file.write_dataset("mean_value",mean_value);
 
 
 <h3> Using R and ggplot2 to generate plots</h3>
+@note Alternatively, one could use the associated python script `plotting.py`.
```

Comment:
What would you think of just adding another section, say `<h3> Using Python and matplotlib to generate plots </h3>` below the R section? In that section, you could add your script and some explanatory text in the same style as was done for R here.